platform/broadcom/sonic-platform-modules-inventec/debian/platform-modules-d6332.init

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+### BEGIN INIT INFO
+# Provides:          setup-board
+# Required-Start:
+# Required-Stop:
+# Should-Start:
+# Should-Stop:
+# Default-Start:     S
+# Default-Stop:      0 6
+# Short-Description: Setup Inventec d6332 board.
+### END INIT INFO
+
+PLATFORM_DIR=/usr/share/sonic/device/x86_64-inventec_d6332-r0/plugins
+
+PLATFORM_DAEMON=$PLATFORM_DIR/platfmgr.py
+PLATFORM_DAEMON_NAME=platfmgr
+
+# The process ID of the script when it runs is stored here:
+PLATFORM_PIDFILE=/var/run/$PLATFORM_DAEMON_NAME.pid
+
+do_monitor_start() {
+    /sbin/start-stop-daemon --quiet --oknodo --pidfile $PLATFORM_PIDFILE --make-pidfile --startas $PLATFORM_DAEMON --start --background -- $DAEMON_OPTS
+}
+
+do_monitor_stop() {
+    /sbin/start-stop-daemon --quiet --oknodo --stop --pidfile $PLATFORM_PIDFILE --retry 10
+}
+
+case "$1" in
+start)
+	echo -n "Setting up board... "
+	
+#	depmod -a
+	/usr/local/bin/inventec_d6332_util.py -f install
+        do_monitor_${1}
```

Comment:
Inconsistent alignment in this file. Please replace all tabs with 4 spaces; each level on indent should be 4 spaces.

Reply:
@jleveque I have replaced tab with 4 spaces in this file and submitted a new commit. Thanks a lot for the comments.